doc/whats_new/v0.21.rst

```diff
@@ -71,6 +71,11 @@ Support for Python 3.4 and below has been officially dropped.
 :mod:`sklearn.cluster`
 ......................
 
+- |API| Deprecate ``n_components_`` attribute of :class:`AgglomerativeClustering`
+  and :class:`FeatureAgglomeration`. The ``n_components_`` attribute can be
```

Comment:
Needs to be ``` :class:`cluster.AgglomerativeClustering` ``` and ``` :class:`cluster.FeatureAgglomeration` ```

Reply:
No problem, changed in commit [#31354c3](https://github.com/scikit-learn/scikit-learn/pull/13427/commits/4cbe2c16aa586b546ec0b12edf5f0d3f7eddadf1)